tests/unit/drm_plugin_test.sh

```diff
@@ -225,16 +233,99 @@ function test_gui_control_remote()
   compare_command_sequence '' "$LINENO" 'expected_cmd_seq' "$output"
 }
 
+function test_gui_control_after_reboot_local()
+{
+  local gui_on_after_reboot_cmd='systemctl set-default graphical.target'
+  local gui_off_after_reboot_cmd='systemctl set-default multi-user.target'
+  local bind_cmd='for i in /sys/class/vtconsole/*/bind; do printf "%s\n" 1 > $i; done; sleep 0.5'
+  local unbind_cmd='for i in /sys/class/vtconsole/*/bind; do printf "%s\n" 0 > $i; done; sleep 0.5'
+  local output
+
+  # Remote
+  remote_parameters['REMOTE_IP']='127.0.0.1'
+  remote_parameters['REMOTE_PORT']='8888'
+  remote_parameters['REMOTE_USER']='juca'
+
+  tearDown
```

Comment:
Why do we want to call tearDown here?

Reply:
Because we initially want to test the default value of the new commands, but if you look at `setUp` function, it parses a custom config file that changes the default command to new ones, but it's not what we want to test. After that, you can see that i call `parse_configuration "$KW_CONFIG_SAMPLE"`  to test if parsing the config file works as expected. Same code was used in `test_gui_control_remote()` function.